Tools/ros2/ardupilot_msgs/msg/Rally.msg

```diff
@@ -0,0 +1,13 @@
+geographic_msgs/GeoPoint point
+float32 break_altitude # when autolanding, break out of loiter at this alt (meters)
```

Comment:
What if a platform doesn't fly? What should this be set to?

Reply:
that is just a copy/paste from the `AP_Rally` library. You can add rally points at all times.

Reply:
The altitude is respect to Home, added to the comment.

Reply:
@Ryanf55  Question for developers: should we keep this flag in Above home altitude (and if so, I'd like to change name of the flag to `break_altitude_above_home`) or to get the same reference as the Rally altitude (and internally convert it to Above Home Altitude)

Reply:
from dev call: ignore this interface as it is not supported by the code.